cce/cce.go

```diff
@@ -195,22 +209,62 @@ func (h *Handler) post() {
 		handleError(fmt.Sprintf("falha ao remover diretorio temporario criado %s, erro %q", unzipDestination, err), h)
 	}
 	h.Status = status.Idle
+	h.Err = ""
+	log.Println("CCE Status [ IDLE ]")
+}
+
+// save candidates on gcs
+func (h *Handler) saveCandidatesOnGCS(candidates []*descritor.Candidatura) error {
```

Comment:
Quando for salvar no GCS, você não precisa salvar o zip em disco. Pode realizar a compressão todas em memória e mandar os bytes do zip direto para o GCS.